### DISCUSSION TITLE
lps node firmware

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1115] [Q&A]
Hi, I want to upload the new algorithm in the lps node firmware. I found the lps node firmware this file. But which one is algorithm I can modify in the file. Please to help, thanks.

**Comment by knmcguire** [comment]
Hi! 

I'm assuming you found this repo already right? https://github.com/bitcraze/lps-node-firmware/

How about the  `uwb_tdoa_anchor` or `uwb_twr_anchor` files in `src/` ?

**Reply by CalvinKoung** [reply]
copy that, I will try it. tks
